.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/gabebw/hotline-webring
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+    docker:
+    - image: circleci/ruby:2.5.3-node
+      environment:
+        RAILS_ENV: test
+        PGHOST: localhost
+        PGUSER: hotlinewebring
+    - image: circleci/postgres:10.6
+      environment:
+        POSTGRES_USER: hotlinewebring
+        POSTGRES_DB: hotlinewebring_test
+        POSTGRES_PASSWORD: ""
+
+    steps:
+      - checkout
+      - run:
+          name: Create Cache Directories
+          command: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+      - run:
+          name: Create .env
+          command: cp .sample.env .env
+
+      # Restore the dependency cache
+      - restore_cache:
+          keys:
+          - hotlinewebring-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+          - hotlinewebring-
```

Comment:
We could probably remove this line, which is a fallback place to look for the dependency cache.